codex-process-feasibility/README.md

```diff
@@ -0,0 +1,58 @@
+# NUM CODEX Feasibility Process
+
+The feasibility process is used to run a distributed feasibility query. A feasibility query contains multiple criteria and returns one population count per organization. The feasibility query is requested by one organization (usually the Zentrale Antrags- und Registerstelle (ZARS)) and executed in multiple organizations (usually Datenintegrationszentren (DIZ)).
+
+## High-Level Overview
+
+![fig-1](./docs/codex-feasibility-process-01.png)
+
+As you can see in the figure above, the feasibility query process starts with a `request` message (1) that the ZARS or any other initiating organization sends to itself. By sending this message to the ZARS itself, the business process engine (BPE) of the ZARS can be used to distribute the `execute` messages (2) to all DIZ'es in question. After receiving the `execute` messages, each DIZ will calculate the result and answer with a `result` message (3). The ZARS will accumulate the results and provide the requester with live updates.
```

Comment:
Do not describe processes in prose! Use numbered lists instead.
When using numbered lists make sure the content is `parallel`, i.e. **all** list entries are either:
- single words
- full sentences
- short phrases

The important thing is that the reader can recognize them as coherent.

The use of numbered lists is also beneficial for jumping quickly between illustration and explanation. Inline references like (3) are hard to find.

Reply:
Applies to all the other descriptions as well. Has only been vicariously picked.